arrays/arrays.c

```diff
@@ -19,45 +19,70 @@ typedef struct Array {
 /*****
  * Allocate memory for a new array
  *****/
-Array *create_array (int capacity) {
+Array *create_array(int capacity)
+{
   // Allocate memory for the Array struct
+  Array *arr = malloc(sizeof(Array));
 
   // Set initial values for capacity and count
+  arr->capacity = capacity;
+  arr->count = 0;
 
   // Allocate memory for elements
+  arr->elements = malloc(sizeof(char *) * capacity);
```

Comment:
This is good. You could also have used `calloc()` here which zero-initialized. In some environments malloc is initialized with garbage in the allocated memory, which can then be overwritten as needed. It could potentially cause an issue if you expect that a certain array index will have a value of null (e.g. when checking if a certain array index is empty in a conditional statement) but it actually has some other unexpected, non-zero value.